src/EuroDNS/Helper/SimpleLogger.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Upmind\ProvisionProviders\DomainNames\EuroDNS\Helper;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+class SimpleLogger implements LoggerInterface
```

Comment:
Please delete this. A logger will be injected into the Provider instance for you to use. If you arent sure how to view provision request logs using the Provision Workbench, let me know.

Reply:
updated

Reply:
Please implement usage of the PSR-3 logger instance. It looks like a sensible place for this may be the `EuroDNSApi::connect()` method.

Example usage:

```php
if (isset($this->logger)) {
    $this->logger->debug('EuroDNS Request:' . $request);
}

// do curl ...

if (isset($this->logger)) {
    $this->logger->debug('EuroDNS Response:' . $response);
}
```

Reply:
Updated